### ISSUE TITLE
Analytics for app open frequency

### ISSUE THREAD

**Issue by Raymo111** [Deletescape-Media/Lawnchair#4164]
### Describe the feature

I like to only keep the apps I use the most often on the home screen, but these apps change very often. I would like some sort of heat map or open count for apps on the home screen and potentially even in the drawer. Essentially, Lawnchair could keep track of app opens (locally on the device for privacy, of course) and show counts as statistics to users.

### Why would it be useful to add?

This feature gives users the ability to analyze the usage frequency of apps on the home screen, facilitating updating which apps are on the home screen, similar to how the prediction bar in the drawer works, but more transparently.

### Did the feature exist in Lawnchair v2? (Play Store version)

No

### Example(s)

Can be shown in the edit view (when pressing and holding an icon), and/or by some other means. Can be shown by color (heat map) or number (how many opens today/this week/etc - maybe user selectable what time period to show/maintain history of).

### Additional context

Every time an app is opened, its counter in Lawnchair should be incremented, and the counter can be cleared manually and opens will be automatically expired/decremented on a rolling basis after some configurable time.

**Comment by ammargitham** [comment]
[Digital Wellbeing](https://support.google.com/android/answer/9346420?hl=en) already does this.

**Comment by Raymo111** [comment]
Not really, Digital Wellbeing shows how long apps are used for, not how many times they're opened from the home screen.

**Comment by Bruhipie** [comment]
> Not really, Digital Wellbeing shows how long apps are used for, not how many times they're opened from the home screen.

Digital Wellbeing doesnt show the amount of times you opened an app from home screen but it does show total no. of times you opened the app. In your daily screen time stats, click on an app and you will have 2 drop down menus below the app name, select "Times Opened".

**Comment by Raymo111** [comment]
Oh cool, thanks for letting me know that exists. Problem is, that still doesn't show exactly how many times I tap it from the home screen to open it, versus say switching between apps, or from tapping notifications (i.e. I never open Facebook from the home screen but I tap notifications quite often). So this sort of works, but also is still not as perfect as proper stats built into Lawnchair.
